include/sddf/network/lib_sddf_lwip.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2022, UNSW
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <microkit.h>
+#include <sddf/network/queue.h>
+#include "lwip/pbuf.h"
+
+/* Default gigabit link speed. */
+#define SDDF_LWIP_LINK_SPEED 1000000000
+
+/* Default ethernet MTU. */
+#define SDDF_LWIP_ETHER_MTU 1500
+
+/* Definitions for sDDF error constants. */
+typedef enum {
+    /* No error, everything OK. */
+    SDDF_ERR_OK = 0,
+    /* Pbuf too large for sDDF buffer. */
+    SDDF_ERR_PBUF = -1,
+    /* No buffers available. */
+    SDDF_ERR_NO_BUF = -2,
+    /* Pbuf successfully enqueued to be sent later. */
+    SDDF_ERR_ENQUEUED    = -3,
+    /* Could not resolve error. */
+    SDDF_ERR_UNHANDLED    = -4
```

Comment:
The names are a bit too generic when you consider other device classes.

Perhaps a prefix like `SDDF_LWIP_ERR_` would be better.

Reply:
Sure, sounds reasonable